Configs/.local/lib/hyde/wallpaper.waydeeper.sh

```
@@ -0,0 +1,102 @@
+    cat << EOF
+
+Error: Another instance of $(basename "$0") is running.
+If you are sure that no other instance is running, remove the lock file:
+    $lockFile
+EOF
+    exit 1
+fi
+touch "$lockFile"
+trap 'rm -f "${lockFile}"' EXIT
+scrDir="$(dirname "$(realpath "$0")")"
+source "$scrDir/globalcontrol.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./globalcontrol.sh was not specified as input (see shellcheck -x).

(SC1091)